setup.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import codecs
+import os
+from typing import List
+
+from setuptools import setup
+
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding="utf-8").read()
+
+
+install_requires: List[List[str]] = []
+install_requires += (["jsonschema"],)
+
+
+setup(
+    name="ngff",
+    version="0.0.4dev",
+    author="The Open Microscopy Team",
+    url="https://github.com/ome/ngff",
+    description="Next-Generation File Format: spec and schemas",
+    long_description=read("README.md"),
+    long_description_content_type="text/markdown",
+    packages=["ngff"],
```

Comment:
See https://github.com/ome/ome-zarr-py/pull/142#pullrequestreview-846525703 for a suggestion of prefixing the package with `ome_`